CONTRIBUTING.md

```diff
@@ -71,6 +71,24 @@ After importing the project, import the IDE settings as well.
 You can configure your build tool and IDE to add `-parameters` javac option.
 Please refer to [Configure `-parameters` javac option](http://line.github.io/armeria/setup.html#configure-parameters-javac-option) for more information.
 
+### Use JUnit5 instead of JUnit4 for test
+
+We support both JUnit4 and JUnit5 for test. but, we recommend to use JUnit5.
```

Comment:
Could you add links for them? 😁
* https://junit.org/junit4/
* https://junit.org/junit5/

Reply:
Sure! 
I did it.